__tests__/app.test.js

```diff
@@ -118,4 +120,47 @@ describe("GET /api/articles/:article_id", () => {
         expect(response.body).toEqual({ msg: "Article not found" });
       });
   });
+
+  describe("Get /api/articles", () => {
+    test("Get ALL ARTICLES sorted by created by", () => {
+      return request(app)
+        .get("/api/articles/")
+        .then((response) => {
+          const retrievedArticles = response.body;
+          expect(response.status).toBe(200);
+          expect(new Date(retrievedArticles[0].created_at)).toEqual(
+            new Date("2020-11-03T09:12:00.000Z")
+          );
+          for (let i = 1; i < retrievedArticles.length; i++) {
+            expect(
+              new Date(retrievedArticles[i].created_at).getTime()
+            ).toBeLessThanOrEqual(
+              new Date(retrievedArticles[i - 1].created_at).getTime()
+            );
+          }
+        });
+    });
+    test("Articles should have a comment count property", () => {
+      return request(app)
+        .get("/api/articles/")
+        .then((response) => {
+          const retrievedArticles = response.body;
+          retrievedArticles.forEach((article) => {
+            expect(article).toHaveProperty("comment_count");
+          });
+        });
+    });
+
+    test("Articles should NOT have a 'body' property once returned", () => {
+      return request(app)
+        .get("/api/articles/")
+        .then((response) => {
+          const retrievedArticles = response.body;
+          expect(response.status).toBe(200);
+          retrievedArticles.forEach((article) => {
+            expect(article.body).toBe(undefined);
+          });
```

Comment:
I would also like to see a test here which checks the articles have been sorted correctly. Jest sorted would be useful here